config/magpie.ini

```diff
@@ -84,7 +84,7 @@ prefix = /magpie
 
 [server:main]
 use = egg:gunicorn#main
-host = localhost
+host = 0.0.0.0
 port = 2001
```

Comment:
We have the now required environment variable MAGPIE_PORT and it is not used.

Reply:
Yes. I have yet to find a way to simplify this. I have tried defining a "shared" section where `magpie.port`, and `[server:main]`'s `port` would use the same reference, but it was not working.

Reply:
Ok, then I suggest that you add a small comment telling users that port must be defined at both place.

Reply:
I still wonder though why `MAGPIE_PORT` even needs to be defined in your case ?
It is important to remember that ``MAGPIE_PORT``, ``MAGPIE_URL``, etc.  are only variable to set the "exposed" URL. 
So an application running on `http://my-pavics-server.ca` should define exactly that value in `MAGPIE_URL` only. 
The ``port`` of ``[server:main]`` still remains `2001` because it is where the local app is accessible from the docker. 

The ``MAGPIE_PORT`` should only be `2001` when running locally.

Reply:
From my perspective I wonder why you did not saw this error on your side 😜, I guess that MAGPIE_URL is not set properly somewhere. But with https://github.com/Ouranosinc/Magpie/pull/418 I'm not worried anymore. 
I also suggest that you remove MAGPIE_PORT and fix MAGPIE_URL if required in https://github.com/bird-house/birdhouse-deploy/pull/143 before running tests.